.github/workflows/ci.yaml

```diff
@@ -1,3 +1,7 @@
+# THIS FILE WAS AUTOMATICALLY GENERATED, PLEASE DO NOT EDIT.
```

Comment:
this doesn't seems to get updated when I generate with a newer version of kres, not sure why? :thinking:

Reply:
it should be updated only if the file has changes without the header